index.js

```diff
@@ -14,7 +16,35 @@ app.get('/', async (req, res) => {
 });
 
 // TODO change to post when finish testing
-app.get('/create-pdf', async (req, res) => {
+app.get('/certificate', async (req, res) => {
+  if (!req.query.id) {
+    //TODO ERROR HANDLING
+    res.status(400).send('Missing id');
+    return;
+  }
+  //Getting the buyer's name from the query if it exists
+  const nameRu = req.query.nameRu;
+  const nameHe = req.query.nameHe;
+  const chapterDoc = await Chapter.findOne({ id: +req.query.id });
```

Comment:
why +?
use findById

Reply:
findbyid is looking by mongo id and we have our own id, don't we?